homeassistant/components/oru/sensor.py

```diff
@@ -0,0 +1,69 @@
+"""Platform for sensor integration."""
+import logging
+
+from oru import Meter
+from oru import MeterError
+from homeassistant.const import ENERGY_KILO_WATT_HOUR
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_METER_NUMBER = "meter_number"
+
+SENSOR_NAME = "ORU Current Energy Usage"
+SENSOR_ICON = "mdi:counter"
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Please add a PLATFORM_SCHEMA to validate the meter number

Reply:
Done.